### ISSUE TITLE
[PATCH] Twisted overview howto needs updating

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7206]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)| @jml reported|
|-|-|
|Trac ID|trac#481|
|Type|defect|
|Created|2004-01-18 10:41:39Z|

Attachments:

* [twisted-overview.NEW.dia](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/297/2978e0c34dbebfc46bf96c994436a5790a22017a/d038044f8705618b9c390cd52a9d496a807d4f20.dia) (5984 bytes) - added by headly on 2004-10-19 23:14:14Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__481 481
type__defect defect
reporter__jml jml
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__documentation documentation
time__1074422499000000 1074422499000000
changetime__1470794769146020 1470794769146020
version__ 
owner__ 
cc__spiv cc__jml cc__hypatia
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)|@jml commented|
|-|-|

```
#!html
<pre>
The diagram in Twisted overview needs updating. Twisted.Words and
Twisted.Persisted seem to be things of the past.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
Reassigned to spiv, since he did the original diagram. spiv, if you want me to
do it, assign it back.

And is the dia file in CVS?
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|spiv commented|
|-|-|

```
#!html
<pre>
Mary added the comment:
> 
> Reassigned to spiv, since he did the original diagram. spiv, if you want me to
> do it, assign it back.

Well, I can't do it for the next few days -- I don't want to run dia on this
laptop.

If people can wait, I'm happy to redo it.

> And is the dia file in CVS?

Yep -- look in doc/img
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
It has been open for just under a month, it's probably OK to wait until Tuesday...
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
Reassigning
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="headly's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|headly commented|
|-|-|

```
#!html
<pre>
As I was working on this I soon noticed I was maybe adding too much, and not
enough in other areas.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
Thank you very much for the diagram: we have replaced the current one with your
one in the repository.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_9"></a>|GitHub commented|
|-|-|

In changeset d425bbe37b13d87600a6cf28ee8f9e1dcfec4cda
```
#!CommitTicketReference repository="" revision="d425bbe37b13d87600a6cf28ee8f9e1dcfec4cda"
Merge pull request #481 from twisted/8739-rodrigc-runner-py3

Author: rodrigc
Reviewer: alex
Fixes: #8739

Port twisted.runner to Python 3
```
